cheatsheets/Clickjacking_Defense_Cheat_Sheet.md

```diff
@@ -77,6 +77,22 @@ Meta-tags that attempt to apply the X-Frame-Options directive DO NOT WORK. For e
 - **X-Frame-Options Deprecated** While the X-Frame-Options header is supported by the major browsers, it was never standardized and has been deprecated in favour of the frame-ancestors directive from the CSP Level 2 specification.
 - **Proxies** Web proxies are notorious for adding and stripping headers. If a web proxy strips the X-Frame-Options header then the site loses its framing protection.
 
+# Defending with SameSite Cookies
+
+The `SameSite` cookie attribute defined in [RFC 6265bis](https://tools.ietf.org/html/draft-ietf-httpbis-rfc6265bis-02#section-5.3.7) is primarily intended to defend against [cross-site request forgery (CSRF)](Cross-Site_Request_Forgery_Prevention_Cheat_Sheet.md#samesite-cookie-attribute); however it can also provide protection against Clickjacking attacks.
+
+Cookies with a `SameSite` attribute of either `strict` or `lax` will not be included in requests made to a page within an `<iframe>`. This means that if the session cookies are marked as `SameSite`, any Clickjacking attack that requires the victim to be authenticated will not work, as the cookie will not be sent. An article on the [Netsparker blog](https://www.netsparker.com/blog/web-security/same-site-cookie-attribute-prevent-cross-site-request-forgery/) provides further details on which types of requests cookies are sent for with the different SameSite policies.
+
+This approach is discussed on the [JavaScript.info website](https://javascript.info/clickjacking#samesite-cookie-attribute).
+
+## Limitations
+
+As of 2019 the `SameSite` attribute is supported by [most modern browsers](https://caniuse.com/#feat=same-site-cookie-attribute), although it it not widely supported on mobile browsers. Support in Internet Explorer will varies depending on the updates installed and the underlying operating system.
```

Comment:
This is not a limitation. It almost is supported by even all mobile browsers. Could you possibly restructure this sentence?
- will `vary` instead of `varies`

Reply:
I've reworded the sentence to soften it a bit and removed the note about mobile browsers - but according the the caniuse stats there's currently 12% of users with browsers that don't support it, which is non-trivial.

Thoughts on the new wording?